Makefile.dep

```diff
@@ -5,7 +5,8 @@ OLD_USEPKG := $(sort $(USEPKG))
 # include board dependencies
 -include $(RIOTBOARD)/$(BOARD)/Makefile.dep
 
-# pull dependencies from drivers
+# pull dependencies from sys and drivers
+include $(RIOTBASE)/sys/Makefile.dep
```

Comment:
not sure about introducing this here too. Though it is consistent with `/drivers`, but should still go in a separate PR and then adapt all sys modules deps, too.

Reply:
I'm not sure what I should adapt. I could move it into a separate PR, but the only entry in `sys/Makefile.dep` is now Fortuna PRNG related. That would not make sense.

Reply:
I've moved it to `sys/Makefile.dep` because of https://github.com/RIOT-OS/RIOT/pull/5675#discussion_r157827407.

Reply:
okay leave as is, I was just think that other sys-modules could be adapted, i.e., move deps form `RIOT/Makefile.dep` to `RIOT/sys/Makefile.dep` - but obviously not in this PR 😄